Google.PowerShell/Compute/GceCmdlet.cs

```diff
@@ -110,6 +122,51 @@ private void WriteWarnings(Operation op)
             }
         }
 
+        private void WriteProgress(Operation op, string progressMessage)
+        {
+            int activityId = (int)(op.Id % int.MaxValue ?? 0);
```

Comment:
I assume `op.Id` is an int64? What is the `?? 0` needed for? What does that even mean when dealing with integers?